bower.json

```diff
@@ -23,7 +23,10 @@
     "paper-progress": "PolymerElements/paper-progress#^1.0.0",
     "iron-flex-layout": "PolymerElements/iron-flex-layout#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "iron-behaviors": "PolymerElements/iron-behaviors#^1.0.0"
+    "paper-ripple": "PolymerElements/paper-ripple#^1.0.0",
+    "iron-behaviors": "PolymerElements/iron-behaviors#^1.0.0",
+    "paper-behaviors": "PolymerElements/paper-behaviors#^1.0.1",
```

Comment:
@cdata `1.0.1` has the behavior that this code depends on. How do you handle cases like this?